src/generators/net_caffe.cc

```diff
@@ -230,7 +231,41 @@ namespace dd
 	add_embed(this->_dnet_params,top,"embed",input_dim,num_output);
       }
   }
-  
+
+	template <>
+	void NetInputCaffe<ImgTorchInputFileConn>::configure_inputs(const APIData &ad_mllib,
```

Comment:
`ad_mllib` looks unused.

Reply:
mmm yes but these are mandatory for template instantiation, as far as I understand